mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2229,7 +2229,8 @@ static func::FuncOp createGpuGemmKernel(ModuleOp module,
   if (!params.perfConfig.empty())
     gemm->setAttr("perf_config", b.getStringAttr(params.perfConfig));
 
-  gemm->setAttr("split-k-factor", b.getI32IntegerAttr(splitkFactor));
+  if (splitKFactor > 1)
```

Comment:
Why? This'll be part of the perf config?

Reply:
I was thinking that there may be a scenario where one wants to experiment with just matrix sizes and `splitKFactor` without bothering about tiling (via the `perfConfig` string).

What do you think?

Reply:
Agree. Let's remove it

Reply:
done